FBTweak/_FBTweakCollectionViewController.m

```diff
@@ -64,6 +77,24 @@ - (void)viewWillDisappear:(BOOL)animated
   [_keyboardManager disable];
 }
 
+- (BOOL)_supportsUpdates {
+  return [self.tweakCategory respondsToSelector:@selector(updateWithCompletion:)];
+}
+
+- (void)_updateCategory {
+  self.refreshControl.attributedTitle =
+      [[NSAttributedString alloc] initWithString:@"Refreshing..."];
+  [self.refreshControl beginRefreshing];
+  [self.tweakCategory updateWithCompletion:^(NSError * _Nullable error) {
+      dispatch_async(dispatch_get_main_queue(), ^{
+        [self.refreshControl endRefreshing];
+        if (error) {
+          
```

Comment:
Do what? :)

Reply:
I actually didn't know what to do here. Decided to just show an error.